reana/cli.py

```diff
@@ -124,6 +136,22 @@ def cli():  # noqa: D301
         $ reana-cluster -f reana-cluster-latest.yaml init
         $ # we now have REANA cluster running "master" versions of components
 
+    How to install latest ``master`` REANA client CLI script:
+
+    .. code-block:: console
+
+        \b
+        $ workon reana
```

Comment:
I would remove this `workon` and the one below since we never do `deactivate` since the `mkvirtualenv`  in L112